docs/docs/actions.mdx

```diff
@@ -3222,6 +3222,195 @@ Returns a promise that resolves when the clock speed has been set.
 server.rpioSPIEnd();
 ```
 
+### `server.serialConnect()`
+
+Establish the connection to the bluetooth serial device
+The action is only executed if <TagLink tag='@onStoryAction'/> has been configured to perform events in `device` actions.
+
+Returns a promise that resolves on the 'open' event.
+
+The **first parameter** is a string with the path to the device.
+The **second parameter** is an object with the options you want to connect to the device with.
+{boolean} [autoOpen=true] Automatically opens the port on `nextTick`.
+{number=} [baudRate=9600] The baud rate of the port to be opened. This should match one of the commonly available baud rates, such as 110, 300, 1200, 2400, 4800, 9600, 14400, 19200, 38400, 57600, or 115200. Custom rates are supported best effort per platform. The device connected to the serial port is not guaranteed to support the requested baud rate, even if the port itself supports that baud rate.
+{number} [dataBits=8] Must be one of these: 8, 7, 6, or 5.
+{number} [highWaterMark=65536] The size of the read and write buffers defaults to 64k.
+{boolean} [lock=true] Prevent other processes from opening the port. Windows does not currently support `false`.
+{number} [stopBits=1] Must be one of these: 1 or 2.
+{string} [parity=none] Must be one of these: 'none', 'even', 'mark', 'odd', 'space'.
+{boolean} [rtscts=false] flow control setting
+{boolean} [xon=false] flow control setting
+{boolean} [xoff=false] flow control setting
+{boolean} [xany=false] flow control setting
+{object=} bindingOptions sets binding-specific options
+{Binding=} Binding The hardware access binding. `Bindings` are how Node-Serialport talks to the underlying system. Will default to the static property `Serialport.Binding`.
+{number} [bindingOptions.vmin=1] see [`man termios`](http://linux.die.net/man/3/termios) LinuxBinding and DarwinBinding
+{number} [bindingOptions.vtime=0] see [`man termios`](http://linux.die.net/man/3/termios) LinuxBinding and DarwinBinding
+
+#### Examples:
+
+1. Connect to the device '/dev/rfcomm0' with the baudRate option set to 9600.
+```typescript
+await server.serialConnect("/dev/rfcomm0", {
+  baudRate: 9600,
+});
+```
+
+### `server.serialStream()`
+
+Parses and returns the serial stream to the event tag 'onSerialData'.
+The action is only executed if <TagLink tag='@onStoryAction'/> has been configured to perform events in `device` actions.
+
+Shouts data to the onStreamData event tag.
```

Comment:
@WesleyMayle can you add `onStreamData` to listen-tags.mdx? Then you can link to it with the TagLink element like `@onStoryAction`

Reply:
Sure, I'll look at that.